manual/cypher/cypher-docs/src/docs/dev/general/compatibility.asciidoc

```diff
@@ -1,24 +1,19 @@
 [[cypher-compatibility]]
 = Compatibility
 
-Cypher is still changing rather rapidly.
-Parts of the changes are internal -- we add new pattern matchers, aggregators and optimizations or write new <<query-tuning,query planners>>, which hopefully makes your queries run faster.
-
-Other changes are directly visible to our users -- the syntax is still changing.
-New concepts are being added and old ones changed to fit into new possibilities.
-To guard you from having to keep up with our syntax changes, Neo4j allows you to use an older parser, but still gain speed from new optimizations.
-
-There are two ways you can select which parser to use.
-You can configure your database with the configuration parameter `cypher.default_language_version`, and enter which parser you'd like to use (see <<cypher-versions>>)).
-Any Cypher query that doesn't explicitly say anything else, will get the parser you have configured, or the latest parser if none is configured.
+Cypher is not a static language - every new version brings new features and sometimes old features get dropped.
+It is possible to access older language versions if that is needed.
+There are two ways you can select which version to use.
+You can configure your database with the configuration parameter `cypher.default_language_version`, and enter which version you'd like to use (see <<cypher-versions>>)).
+Any Cypher query that doesn't explicitly say anything else, will use the version you have configured, or the latest version if none is configured.
```

Comment:
I suggest the text:
-->
Every Cypher query will use this version, provided the query hasn't explicitly been configured as described in the next item.